### PR TITLE
{Misc} Update local context doc for new name param persist

### PR DESCRIPTION
**Pull request by yungezz**
**Description**

Update local context doc for new name param persist

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to have a look? thanks

File: doc/parampersist_localcontext_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+The document provides instructions and guidelines on how to board param persist feature, prevoius name local context. The public feature name to end user is param persist, usage is at [here](https://docs.microsoft.com/en-us/cli/azure/param-persist-howto). In implementation, it's referened as local context. The 2 terms are equivalent in this doc.
```
Comment (fengzhou-msft) [inline doc/parampersist_localcontext_guidelines.md:1]:
```suggestion
The document provides instructions and guidelines on how to board param persist feature, prevoius name local context. The public feature name to end user is param persist, usage is at [here](https://docs.microsoft.com/cli/azure/param-persist-howto). In implementation, it's referenced as local context. The 2 terms are equivalent in this doc.
```

File: doc/parampersist_localcontext_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+The document provides instructions and guidelines on how to board param persist feature, prevoius name local context. The public feature name to end user is param persist, usage is at [here](https://docs.microsoft.com/cli/azure/param-persist-howto). In implementation, it's referenced as local context. The 2 terms are equivalent in this doc.
```
Comment (qwordy) [inline doc/parampersist_localcontext_guidelines.md:1]:
```suggestion
The document provides instructions and guidelines on how to board param persist feature, prevoius name local context. The public feature name to end user is param persist. The usage is at [here](https://docs.microsoft.com/cli/azure/param-persist-howto). In implementation, it's referenced as local context. The 2 terms are equivalent in this doc.
```

**Review by qwordy — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**